client/List.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+class Item extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {isCompleted: false};
+        this.handleClick = this.handleClick.bind(this);
+    }
+    handleClick(){
+        this.setState({isCompleted: !this.state.isCompleted});
+    }
+    render(){
+        const {name} = this.props;
+        const itemWithStatus = this.state.isCompleted ? <strike>{name}</strike> : name; 
+        return (
+        <span onClick={this.handleClick}>
```

Comment:
IMO each list item should be a button with a handleClick.  You don't want to use `span` for click event handling, because it's less accessible.

Reply:
aah I see